paragon/_account.scss

```
@@ -0,0 +1,324 @@
+.page__account-settings {
+  max-width: 1600px;
+  margin: 0 auto;
+  padding: 35px 15px 20px !important;
+  .form-control {
+    padding: 6px 10px;
+    font-size: 14px;
+    line-height: 20px;
+    height: 32px;
+    &:hover,
+    &:focus {
+      box-shadow: none;
+      background-color: $concrete;
```

This wasn't expected and previously unkown to me.

But it turns out that setting colors like this degrades our MFE setup.

Please use CSS variabes:
Somewhere on the top of the page, it should have all variables as CSS variales:
```
:root {
    --fx-concrete-color: $concrete;
}
```
In the individual sytles, it should use the CSS variable so it's customizable in runtime.
```suggestion
      background-color: var(--fx-concrete-color);
```